pkg/flow/adapter/xslttransform/adapter.go

```diff
@@ -57,7 +58,8 @@ func NewTarget(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClien
 		targetce.ReplierWithStatefulHeaders(env.BridgeIdentifier),
 		targetce.ReplierWithStaticDataContentType(cloudevents.ApplicationXML),
 		targetce.ReplierWithStaticErrorDataContentType(*cloudevents.StringOfApplicationJSON()),
-		targetce.ReplierWithPayloadPolicy(targetce.PayloadPolicy(targetce.PayloadPolicyAlways)))
+		targetce.ReplierWithPayloadPolicy(targetce.PayloadPolicy(targetce.PayloadPolicyAlways)),
+		targetce.ReplierWithStaticResponseType(v1alpha1.EventTypeXSLTTransformError))
```

Comment:
If the event is successful, the incoming event type is used for the outgoing event. IF there is an error you get `io.triggermesh.xslt.transform.error`

Reply:
I would suggest to create tests to make sure that code is behaving as you say.
Also, a component replying the same type that it ingests would lead to loops when replying to sender, I don't think replying the same type should be the default behavior.

Reply:
you got it! I moved your example into a `replier` folder and created a new one in a `sink` folder. are you okay with this?

Reply:
That line is setting a response type `v1alpha1.EventTypeXSLTTransformError` for all resquests, which I don't think is what you want to do.

This is how the component works:

- responses are typed with the incoming type plus the `.response` suffix
- on success they are added the attribute `category: success`.
- on error they are added the attribute `category: error`.